Bottom: 28c8382e4cb651966a554884d99582318f0d4173
Top:    27df1135de7bcc9472c9047db6513e1d8daf4676
Author: Lan Yixun (dlan) <dennis.yxun@gmail.com>
Date:   2013-07-23 17:49:02 +0800

autofs-5.0.7 - fix dead LDAP symbolic link when LDAP support is disabled

autofs will create symbol link mandatory no matter ldap support
is enabled or not. so, without this patch, lookup_ldaps.so will become
a dead link.


---

diff --git a/CHANGELOG b/CHANGELOG
index c328cce..f471f58 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -60,6 +60,7 @@
 - fix typo forced-shutdown should be force-shutdown.
 - fix hesiod check error and use correct $(LIBS) setting.
 - fix compilation of lookup_ldap.c without sasl.
+- fix dead LDAP symbolic link when LDAP support is disabled.
 
 25/07/2012 autofs-5.0.7
 =======================
diff --git a/modules/Makefile b/modules/Makefile
index 2589ae0..43c8a8d 100644
--- a/modules/Makefile
+++ b/modules/Makefile
@@ -76,7 +76,9 @@ install: all
 	-rm -f $(INSTALLROOT)$(autofslibdir)/mount_smbfs.so
 	ln -fs lookup_file.so $(INSTALLROOT)$(autofslibdir)/lookup_files.so
 	ln -fs lookup_yp.so $(INSTALLROOT)$(autofslibdir)/lookup_nis.so
+ifeq ($(LDAP), 1)
 	ln -fs lookup_ldap.so $(INSTALLROOT)$(autofslibdir)/lookup_ldaps.so
+endif
 	ln -fs mount_nfs.so $(INSTALLROOT)$(autofslibdir)/mount_nfs4.so
 ifeq ($(EXT2FS), 1)
  ifeq ($(EXT3FS), 1)
